app/entry.server.tsx

```diff
@@ -132,7 +134,7 @@ function serveBrowsers(...args: DocRequestArgs) {
     remixContext,
     loadContext,
   ] = args
-  const nonce = loadContext.cspNonce ? String(loadContext.cspNonce) : undefined
+  const nonce = String(loadContext.cspNonce)
```

Comment:
I'm not sure I understand why this file was changed.

Reply:
I changed it because Eslint was giving some warning here. 
Changed it back to original now.